src/components/avatar/__stories__/index.story.jsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { Text, Avatar } from '@/index';
+
+export default {
+  title: 'Components/Avatar',
+  component: Avatar,
+  parameters: {
+    docs: {
+      docPage: {
+        a11yProps: ` 
+        aria-label: Add \`aria-label='Open'\` to describe the action of button 
+         `,
+      },
+    },
+  },
+};
+
+export const Basic = () => <Avatar text="JD" appearance="primary" />;
+
+export const Appearance = () => {
+  const weight = 'strong';
+
+  const appearances = [
+    'primary',
+    'secondary',
+    'alert',
+    'warning',
+    'success',
+    'accent1',
+    'accent2',
+    'accent3',
+    'accent4',
+  ];
+
+  return (
+    <div className="d-flex">
+      {appearances.map((appearance, index) => {
+        return (
+          <div key={index} className="mr-9">
+            <Text weight={weight}>{appearance.charAt(0).toUpperCase() + appearance.slice(1)}</Text>
+            <br />
+            <br />
+            <Avatar text="JD" appearance={appearance} />
+          </div>
+        );
+      })}
+    </div>
+  );
+};
+
+export const Size = () => {
+  const weight = 'strong';
+
+  return (
+    <div className="d-flex">
+      <div className="mr-9 d-flex flex-column">
+        <Text weight={weight}>Regular</Text> <br />
+        <Avatar text="JD" size="regular" />
+      </div>
+      <div className="mr-9  d-flex flex-column">
+        <Text weight={weight}>Tiny</Text> <br />
+        <Avatar size="tiny" text="JD" />
+      </div>
+    </div>
+  );
+};
+
+export const AvatarText = () => {
+  const weight = 'strong';
+
+  return (
+    <div className="d-flex">
+      <div className="mr-9  d-flex flex-column">
+        <Text weight={weight}>Initials</Text> <br />
+        <Avatar text="JD" />
+      </div>
+      <div className="mr-9  d-flex flex-column">
+        <Text weight={weight}>Name</Text> <br />
+        <Avatar text="Doe" />
+      </div>
+    </div>
+  );
+};
+
+export const AvatarIcon = () => {
+  const weight = 'strong';
+
+  return (
+    <div className="d-flex">
+      <div className="mr-9  d-flex flex-column">
+        <Text weight={weight}>Tiny</Text> <br />
+        <Avatar size="tiny" icon="person" />
+      </div>
+
+      <div className="mr-9  d-flex flex-column">
+        <Text weight={weight}>Regular</Text> <br />
+        <Avatar icon="person" />
+      </div>
+    </div>
+  );
+};
+
+export const Image = () => {
+  const weight = 'strong';
+
+  return (
+    <div className="d-flex">
+      <div className="mr-9  d-flex flex-column">
+        <Text weight={weight}>Regular</Text> <br />
+        <Avatar appearance="secondary" image={{ src: '/logo.png', altText: 'avatar regular' }} />
+      </div>
```

Comment:
Please update the story. Images are not visible in stories

Reply:
https://mui.com/components/avatars/#letter-avatars

Use first character of alt text and do not show broken image if url is not valid.
If no alt text or image use default icon